data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OffsetManager.java

```diff
@@ -22,11 +22,7 @@
 import io.vertx.core.CompositeFuture;
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
-import java.util.Arrays;
-import java.util.BitSet;
-import java.util.Collection;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
buu.
let's NOT do that

Reply:
That comes from the code formatter

Reply:
I guess this came from the IDE. Reverted it...